service/app.rb

```diff
@@ -59,6 +64,7 @@
 
   measure.source_data_criteria = data_criteria
   measure.cms_id = measure.cms_id.nil? ? 'CMS0v0' : measure.cms_id
+  measure.hqmf_id = madie_measure["id"]
```

Comment:
We should move this to the mapper in qdm-service at some point instead to try and keep the conversion all in one place